app/src/main/java/com/iron/espresso/presentation/home/mystudy/studydetail/ModifyStudyViewModel.kt

```diff
@@ -0,0 +1,119 @@
+package com.iron.espresso.presentation.home.mystudy.studydetail
+
+import android.net.Uri
+import androidx.hilt.lifecycle.ViewModelInject
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import com.iron.espresso.Logger
+import com.iron.espresso.ValidationInputText
+import com.iron.espresso.base.BaseViewModel
+import com.iron.espresso.data.model.*
+import com.iron.espresso.domain.repo.StudyRepository
+import com.iron.espresso.ext.Event
+import com.iron.espresso.ext.networkSchedulers
+import com.iron.espresso.ext.plusAssign
+import com.iron.espresso.ext.toErrorResponse
+import com.iron.espresso.model.api.ModifyStudyRequest
+import retrofit2.HttpException
+
+class ModifyStudyViewModel @ViewModelInject constructor(private val studyRepository: StudyRepository) :
+    BaseViewModel() {
+
+    private val _localItem = MutableLiveData<LocalItem>()
+    val localItem: LiveData<LocalItem>
+        get() = _localItem
+
+    private val _emptyCheckMessage = MutableLiveData<Event<ValidationInputText>>()
+    val emptyCheckMessage: LiveData<Event<ValidationInputText>>
+        get() = _emptyCheckMessage
+
+    private val _success = MutableLiveData<Event<Unit>>()
+    val success: LiveData<Event<Unit>>
+        get() = _success
+
+    val image = MutableLiveData<Uri>()
+
+    fun setStudyImage(image: Uri) {
+        this.image.value = image
+    }
+
+    fun addItems(localItem: LocalItem?) {
```

Comment:
```suggestion
    fun addItem(localItem: LocalItem?) {
```
단일 item이니 단수형이 적합할 것 같고, add면 더해준다는 의미인데, 동작은 다시호출되면 갈아 끼워지기 때문에
registerItem, replaceItem 등의 네이밍을 고민해보면 좋을 것 같네요

Reply:
수정하였습니다
b564064